src/utils/location.js

```diff
@@ -21,7 +21,7 @@ export function createLocation({
 }: IncompleteLocation): SourceLocation {
   return {
     sourceId,
-    line: line || 0,
+    line,
```

Comment:
Or should this be 1 by default. As line number starts with 1, and cannot be < 1.